subworkflows/local/variant_evaluation.nf

```diff
@@ -0,0 +1,49 @@
+//
+// A subworkflow to evaluate variants using truth vcfs.
+//
+
+include { RTGTOOLS_VCFEVAL                 } from '../../modules/nf-core/rtgtools/vcfeval/main'
+include { BCFTOOLS_REHEADER                } from '../../modules/nf-core/bcftools/reheader/main'
+include { TABIX_TABIX as TABIX_TRUTHVCF    } from '../../modules/nf-core/tabix/tabix/main'
+
+workflow VARIANT_EVALUATION {
+
+    take:
+        ch_snv_vcf_tbi     // channel: [mandatory] [ val(meta), path(vcf), path(tbi) ]
+        ch_genome_fai      // channel: [mandatory] [ val(meta), path(fai) ]
+        ch_rtg_truthvcfs   // channel: [mandatory] [ val(meta), path(dbs) ]
+        ch_sdf             // channel: [mandatory] [ val(meta), path(sdf) ]
+
+    main:
+        ch_versions = Channel.empty()
+
+        ch_rtg_truthvcfs
+            .splitCsv ( header:true )
+            .map { row ->
+                return [[samplename:row.samplename[0], bedregions:row.bedregions[0]], row.vcf[0], []]
+            }
+            .set { ch_rtgvcfs_dbs }
+
+        BCFTOOLS_REHEADER (ch_rtgvcfs_dbs, [[:],[]])
+
+        TABIX_TRUTHVCF (BCFTOOLS_REHEADER.out.vcf)
+
+        BCFTOOLS_REHEADER.out.vcf
+            .join(TABIX_TRUTHVCF.out.tbi)
+            .set { ch_truthvcf_tbi }
+
+        ch_snv_vcf_tbi
```

Comment:
Don't we need to split `ch_snv_vcf_tbi` into sample vcfs?

Reply:
Not necessary actually.. specifying --samples <samplename> in the command takes care of that :)

Reply:
oh nice!